.golangci.example.yml

```diff
@@ -1134,52 +1308,283 @@ linters-settings:
+    # Default: false
     allow-trailing-comment: false
+
+    # Can be set to force trailing newlines at the end of case blocks to improve readability.
+    # If the number of lines (including comments) in a case block exceeds this number
+    # a linter error will be yielded if the case does not end with a newline.
+    # Default: 0
     force-case-trailing-whitespace: 0
+
+    # Enforces that an `if` statement checking an error variable is cuddled
+    # with the line that assigned that error variable.
+    # Default: false
     force-err-cuddling: false
+
+    # Enforces that an assignment which is actually a short declaration (using `:=`)
+    # is only allowed to cuddle with other short declarations, and not plain assignments, blocks, etc.
+    # This rule helps make declarations stand out by themselves, much the same as grouping var statement.
+    # Default: false
     force-short-decl-cuddling: false
+
+    # Controls if the checks for slice append should be "strict"
+    # in the sense that it will only allow these assignments to be cuddled with variables being appended.
+    # Default: true
     strict-append: true
 
   # The custom section can be used to define linter plugins to be loaded at runtime.
-  # See README doc for more info.
+  # See README documentation for more info.
   custom:
     # Each custom linter should have a unique name.
-     example:
-      # The path to the plugin *.so. Can be absolute or local. Required for each custom linter
+    example:
+      # The path to the plugin *.so. Can be absolute or local.
+      # Required for each custom linter.
       path: /path/to/example.so
-      # The description of the linter. Optional, just for documentation purposes.
+      # The description of the linter.
+      # Optional.
       description: This is an example usage of a plugin linter.
-      # Intended to point to the repo location of the linter. Optional, just for documentation purposes.
+      # Intended to point to the repo location of the linter.
+      # Optional.
       original-url: github.com/golangci/example-linter
 
 linters:
+  # Disable all linters.
   disable-all: true
+  # Enable specific linter
+  # https://golangci-lint.run/usage/linters/#enabled-by-default-linters
   enable:
-    - megacheck
+    - asciicheck
+    - bidichk
+    - bodyclose
+    - contextcheck
+    - cyclop
+    - deadcode
+    - depguard
+    - dogsled
+    - dupl
+    - durationcheck
+    - errcheck
+    - errname
+    - errorlint
+    - exhaustive
+    - exhaustivestruct
+    - exportloopref
+    - forbidigo
+    - forcetypeassert
+    - funlen
+    - gci
+    - gochecknoglobals
+    - gochecknoinits
+    - gocognit
+    - goconst
+    - gocritic
+    - gocyclo
+    - godot
+    - godox
+    - goerr113
+    - gofmt
+    - gofumpt
+    - goheader
+    - goimports
+    - golint
+    - gomnd
+    - gomoddirectives
+    - gomodguard
+    - goprintffuncname
+    - gosec
+    - gosimple
     - govet
+    - ifshort
+    - importas
```

Comment:
`no-unaliased` of `importas` is `false` by default, yet the config says it's `true`.

Reply:
the config is not the default value, the comments contain the default value.